src/python/grpcio/grpc/aio/_typing.py

```diff
@@ -36,7 +36,10 @@
 DeserializingFunction = Callable[[bytes], Any]
 MetadatumType = Tuple[MetadataKey, MetadataValue]
 MetadataType = Union[Metadata, Sequence[MetadatumType]]
-ChannelArgumentType = Sequence[Tuple[str, Any]]
+ChannelArgumentType = Tuple[Union[str, bytes], Any]
```

Comment:
What I mentioned above about public methods make this change impossible as well.